pkg/serverless/trace/span_modifier.go

```diff
@@ -39,7 +40,12 @@ func (s *spanModifier) ModifySpan(_ *pb.TraceChunk, span *pb.Span) {
 
 	// ensure all spans have tag _dd.origin in addition to span.Origin
 	if origin := span.Meta[ddOriginTagName]; origin == "" {
-		traceutil.SetMeta(span, ddOriginTagName, ddOriginTagValue)
+		ddOriginValue := ddOriginTagValueLambda
+		if serverlessInitOrigin := cloudservice.GetCloudServiceType().GetOrigin(); serverlessInitOrigin != "local" {
```

Comment:
Right now we're going to evaluate this conditional on every single span without an origin. This can be quite large. I don't expect that `GetOrigin()` will change after initialization, no?
1. Is this call expensive? Or can it be inlined by the compiler?
2. If it's expensive, can we cache the result or make it a constant set at initialization time?

Reply:
Ah yeah, good point. The origin shouldn't change so it would be much better to set this at init time when we start the tracer